### PR TITLE
[MS] Use MsTextInputModal for invitation link and user email

### PR DESCRIPTION
**Pull request by Max-7**
- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes

### CODE REVIEW

File: client/src/components/core/ms-modal/MsTextInputModal.vue [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <template>
   <ms-modal
     :title="$props.title"
+    :subtitle="$props.subtitle"
```
Comment (NicoTuxx) [inline client/src/components/core/ms-modal/MsTextInputModal.vue:6]: why do you need to use $props, it should work with only subtitle.

**Review by NicoTuxx — DISMISSED** [review]

File: client/src/components/core/ms-modal/MsTextInputModal.vue (hunk 1 above)
Comment (Max-7) [reply client/src/components/core/ms-modal/MsTextInputModal.vue:6]: Yeah it does but I like to be explicit. I don't know what happens if I create a variable in my component named `subtitle`. This way it's explicit, I know where the variable is from, and I avoid potential conflicts.

Comment (NicoTuxx) [reply client/src/components/core/ms-modal/MsTextInputModal.vue:6]: .

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/components/core/ms-modal/MsTextInputModal.vue (hunk 1 above)
Comment (Max-7) [reply client/src/components/core/ms-modal/MsTextInputModal.vue]: Removed the `$props`, see #5488

**Review by NicoTuxx — APPROVED** [review]

**Pull request merged**